puppeteer/bundleRequests.test.js

```diff
@@ -0,0 +1,65 @@
+import puppeteer from 'puppeteer';
+import cypressConfig from '../cypress/support/config';
+
+let browser;
+let page;
+let requests = [];
+
+const config = cypressConfig('local');
+
+const services = [
```

Comment:
Why is this duplicated here, we have this config already in config files can't we import it in rather than duplicate values that will mean we have to update many files?

Reply:
>we have this config already in config files
Do we? Do you mean importing both config and `worldServices`? did it this way as a stop gap untill we merge the configs

Reply:
That just means it's in 2 places though worldServices and here. Seems better to use worldServices.

PS is there an issue for merging the config and worldServices?

Reply:
https://github.com/bbc/simorgh/issues/2131